python/dgl/contrib/sampling/sampler.py

```diff
@@ -509,6 +510,13 @@ class EdgeSampler(object):
     The sampler returns EdgeSubgraph, where a user can access the unique head nodes
     and tail nodes directly.
 
+    This sampler also allows us sample positive edges and negative edges (actually, 
+    nodes) according to their weight. When doing sample we can provide two tensors 
+    representing the weight of each edges and weight of each nodes respectively 
+    should be provided. These weights are used to decide the posibility of 
```

Comment:
How about this?

This sampler allows to non-uniformly sample positive edges and negative edges. For non-uniformly sampling positive edges, users need to provide an array of m elements (`m` is the number of edges), each of which represents the sampling probability of an edge.  For non-uniformly sampling negative edges, users need to provide an array of n elements and the sampler samples nodes based on the sampling probability to corrupt a positive edge.